databricks_cli/sdk/api_client.py

```diff
@@ -105,6 +105,12 @@ def __init__(self, user=None, password=None, host=None, token=None,
         self.api_version = api_version
         self.jobs_api_version = jobs_api_version
 
+    def get_v1_client(self):
```

Comment:
this will make OAuth (and AAD, and OIDC) implementation fragile for long-running commands, where token might have to be refreshed couple of times to keep working. could we have another way of propagating `/api/1.2` path prefix? kwarg in `perform_query` method, for example.

Reply:
I'll change it to use kwargs, that's indeed cleaner